signal/signal.go

```diff
@@ -53,30 +25,33 @@ type test struct {
 // an error if encountered.
 func executeRunner(c config.Config) error {
 	log.Info("==> STARTING SIGNAL RUNNER")
-	healthURL := fmt.Sprintf("http://%s:%d", c.HealthHost, c.HealthAPIPort)
 
-	healthReport, err := pullHealthReport(healthURL, c.HealthEndpoint)
-	if err != nil {
-		log.Error("==> ERROR GETTING REPORT.")
-		log.Error("Are you sure the URL, endport and port are correct?")
+	var (
+		diagnostics = Diagnostics{
```

Comment:
Future work will include a `Cosmos` and a `Mesos` which implement the `Reporter` interface.